addons/auth_totp/static/src/scss/auth_totp.scss

```diff
@@ -0,0 +1,11 @@
+.o_auth_2fa_btn {
```

Comment:
Mmmh ?

Reply:
Gni, actually this is happening in a form view ? You know what "they" say: no CSS on form view, should stay standard.

Reply:
Either it's that, either I need to create a full new widget, (for 5 lines of CSS, meh') either we don't do the custom button to call enable/disable actions at all. @LNAodoo

Reply:
Maybe name file ``res_users_view_form.scss`` so that we know its purpose. Otherwise can't it be replaced by something like ``class="h4"`` to display it a bit bigger ?